<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPatternDialog</class>
 <widget class="QDialog" name="addPatternDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add/Edit Pattern</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Indicator 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>87</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Indicator 2:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelNewPatternButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finishNewPatternButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>51</y>
     <width>87</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern Name:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>127</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="patternNameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="indicator1LineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="indicator2lineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
